app/views/templates/playerControlsExtended.jst

```diff
@@ -1,4 +1,4 @@
-<button id="shuffle">
+<button id="shuffle" <% if (shuffle) { %> class="active" <% } %>>
```

Comment:
ProTip: there's an aria state for that :wink: : [aria-pressed](https://www.w3.org/TR/wai-aria/states_and_properties#aria-pressed)